ko.json

```diff
@@ -0,0 +1,296 @@
+        },
+        "account" : {
+            "home" : {
+                "title" : "계정 정보",
+                "error" : "계정 데이터를 로드하는 중 오류가 발생하였습니다. 잠시 후에 다시 시도하세요.",
+                "info" : {
+                    "title" : "정보",
+                    "line1" : "[provider]를 통해 [username]에 로그인 하였습니다.",
+                    "line2-api-yes" : "[api 접근 권한](/api/docs)이 있습니다.",
+                    "line2-api-no" : "api 접근 권한이 없습니다."
+                },
+                "migration" : {
+                    "title" : "기존 계정",
+                    "waiting-line1" : "로그인 방식이 디스코드로 변경되었습니다.",
+                    "waiting-line2" : "2018년 이전에 만든 계정이 있다면, 이 계정과 병합할 수 있습니다.",
+                    "twitter-line1" : "우리는 디스코드 로그인 방식으로 변경하였습니다. 귀하의 계정은 향후 중단될 수 있는 기존 계정을 이용하고 있습니다.",
+                    "twitter-line2" : "디스코드 계정으로 전환하려면 아래 버튼을 클릭하세요."
+                },
+                "ratings" : {
+                    "title" : "등급",
+                    "none" : "보유한 등급이 없습니다."
+                },
+                "destroy" : {
+                    "title" : "계정 삭제",
+                    "line1" : "더 이상 계정을 이용하지 않고 모든 데이터를 삭제하려면 [여기](/account/destroy)를 클릭하세요."
+                }
+            },
+            "oauth" : {
+                "title" : "fnbr.co에 로그인",
+                "discord-button" : "디스코드로 로그인",
+                "legacy-text" : "혹은, 2018년 이전에 생성한 계정이 있는 경우:",
+                "twitter-button" : "트위터로 로그인",
+                "privacy-text" : "로그인함으로써 [개인정보처리방침](/privacy)에 동의하며, 명시된대로 계정과 관련된 데이터를 저장하는데 동의합니다."
+            },
+            "username" : {
+                "title" : "fnbr.co에 오신것을 환영합니다.",
+                "line1" : "디스코드로 로그인 하였습니다. 사이트의 다른 사용자와 상호작용하려면 계정 이름을 생성해야 합니다.",
+                "nameTaken" : "계정 이름 [attemptedName]을(를) 다른 사용자가 이미 사용중입니다. 디스코드 태그를 사용하는 것을 추천합니다.",
+                "randomName" : "계정 이름 [attemptedName]을(를) 다른 사용자가 이미 사용중입니다. 디스코드 태그를 사용할 수 없었습니다.",
+                "form-username" : "계정 이름",
+                "form-criteria" : "2-32자의 계정 이름을 생성하세요. 영문자 및 숫자, 하이픈(-)과 밑줄(_)을 포함할 수 있습니다.",
+                "button-text" : "계정 이름 설정"
+            },
+            "locked" : {
+                "title" : "로그인 실패",
+                "line1" : "fnbr.co에 로그인할 수 없습니다: [귀하의 계정이 잠겼습니다.](class=text-danger)."
+            },
+            "migrated" : {
+                "title" : "병합된 계정",
+                "line1" : "트위터로 로그인하셨습니다.",
+                "line2" : "그러나, 이 계정은 디스코드로 병합되었습니다."
+            },
+            "notFound" : {
+                "title" : "계정을 찾을 수 없음",
+                "line1" : "트위터로 로그인하셨습니다.",
+                "line2" : "그러나, 이 계정은 등록된 계정이 아니며 기존 계정 병합 목적의 로그인만 지원합니다.",
+                "line3" : "또는, [디스코드로 로그인](/oauth/discord) 할 수 있습니다."
+            }
+        },
+        "errors" : {
+            "404" : {
+                "title" : "에러",
+                "line1" : "요청한 리소스가 없거나, 볼 수 있는 권한이 없습니다."
+            },
+            "500" : {
+                "title" : "Uh oh!",
```

Comment:
Just missed this one.

Reply:
This line is not translated as it is used a lot in Korea.